src/CakeAdapter.php

```diff
@@ -344,15 +344,15 @@ public function fetchAll($sql)
     }
 
     /**
-     * Inserts data into the table
+     * Inserts data into a table.
      *
      * @param Table $table where to insert data
-     * @param array $columns column names
-     * @param $data
+     * @param array $row
+     * @return void
```

Comment:
@dereuromark is that a PHP PSR? I've got `@return void` references everywhere in Phinx.